BENQI08 Updates (#277)
NethermindEth/Forta-Agents: Benqi-Agents/Large-Mint-Burn/README.md

## Test Data

> There are no transactions including large Mints/Burns on PGL contract. We have deployed a PoC contract to emit the monitored events. The bot behaviour can be verified with the following test transactions (Avalanche testnet):

Reviewer: ```suggestion
There are no transactions including large Mints/Burns on PGL contract. There is a PoC contract deployed to emit the monitored events. The bot behaviour can be verified with the following test transactions (Avalanche testnet):
```
Let avoid this `we`
